lark/parsers/lalr_analysis.py

```diff
@@ -279,9 +282,97 @@ def compute_lalr1_states(self):
         else:
             self.parse_table = IntParseTable.from_ParseTable(_parse_table)
 
+    # Generation of conflict resolution hints
+    def __build_reduce_reduce_error(self, la, rules):
```

Comment:
dunder `__names` are not used in Lark. There is not really a reason to used name mangling, and it will probably just create problems down the road with subclassing/testing.

Reply:
It's now fixed